fr/integrations/plugin-packs/procedures/hardware-storage-lenovo-iomega-snmp.md

```diff
@@ -0,0 +1,182 @@
+| core.cpu.utilization.percentage  | CPU Core utilization. | %    |
+
+<!--Interfaces-->
+
+| Metric name                              | Description                                             | Unit |
+|:---------------------------------------- |:------------------------------------------------------- |:---- |
+| status                                   | Status of the interface                                 |      |
+| interface.traffic.in.bitspersecond       | Incoming traffic going through the interface.           | B/s  |
+| interface.traffic.out.bitspersecond      | Outgoing traffic going through the interface.           | B/s  |
+| interface.packets.in.error.percentage    | Incoming errored packets going through the interface.   | %    |
+| interface.packets.in.discard.percentage  | Incoming discarded packets going through the interface. | %    |
+| interface.packets.out.error.percentage   | Outgoing errored packets going through the interface.   | %    |
+| interface.packets.out.discard.percentage | Outgoing discarded packets going through the interface. | %    |
+
+<!--Hardware-->
+
+* Per intances :
+| Metric name                   | Description               | Unit |
+|:----------------------------- |:------------------------- |:---- |
+| disk.status                   | Status of the disk        |      |
+| raid.status                   | Status of the raid        |      |
+| fan.status                    | Status of the fan         |      |
+| temperature.status            | Status of the temperature |      |
+| voltage.status                | Status of the voltage     |      |
+| hardware.fan.speed.rpm        | Speed of fan              | rpm  |
+| hardware.temperature.celsius  | Hardware temperature      | C    |
+| hardware.voltage.millivolt    | Hardware voltage          | mV   |
+
+<!--Memory-->
+
+| Metric name              | Description                | Unit |
+| :----------------------- | :------------------------- |:---- |
+| memory.usage.bytes       | Memory usage               | B    |
+| memory.free.bytes        | Free memory                | B    |
+| memory.usage.percentage  | Memory usage in percentage | %    |
+| memory.buffer.bytes      | Buffer memory              | B    |  
+| memory.cached.bytes      | Memory cached              | B    |  
+
+<!--Storage-->                                                        
+
+| Metric name                            | Description                     | Unit  |
+| :------------------------------------- | :------------------------------ |:----- |
+| storage.partitions.count               | Number of disk partition.       | count |
+| storage.space.usage.bytes              | Used space on a disk partition. | b     |
+| storage.access                         | Access disk partition.          | b     |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+### Configuration Lenovo Iomega 
+
+Afin de contrôler vos équipements Lenovo Iomega, le SNMP v2 doit être configuré.
```

Comment:
There is a link ? Or an official documentation on HOW TO configure SNMP ?

Reply:
User manual added in documentation